labkey/utils.py

```diff
@@ -179,3 +182,18 @@ def handle_response(response, non_json_response=False):
     else:
         # consider response.raise_for_status()
         raise RequestError(response)
+
+
+# Issue #14: json.dumps on datetime throws TypeError
+class DateTimeEncoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, datetime):
```

Comment:
This covers datetimes, but we need handing for date objects since they exhibit the same error.

`isinstance(date.today(), datetime) == false`

Reply:
Good catch. Changed check to `isinstance(o, (datetime, date))`.